apps/site/pages/components/atoms/sr-only.mdx

```diff
@@ -51,7 +77,7 @@ import '@faststore/ui/src/components/atoms/SROnly/styles.scss'
 
 ## Props
 
-<PropsSection name="SROnly" />
+<SROnlyPropsSection component="SROnly" />
```

Comment:
It looks empty here, I think you'll need to add it manually :(
<img width="815" alt="image" src="https://github.com/vtex/faststore/assets/11613011/755e7c2c-f650-4a41-b163-c01a82bf2bf5">